Server/Resources.xcassets/springboard-alerts/springboard-alerts-es.dataset/alerts.json

```diff
@@ -449,5 +449,63 @@
       "OK"
     ],
     "shouldAccept": true
+  },
+  {
+    "title": "¿Permitir que la app %@ use tu ubicación?",
+    "buttons": [
+      "Permitir siempre",
+      "Permitir",
+      "Permitir al usarse la app"
+    ],
+    "shouldAccept": true
+  },
+  {
+    "title": "¿Permitir que la app %@ use tu ubicación aunque no la estés utilizando?",
+    "buttons": [
+      "Permitir"
+    ],
+    "shouldAccept": true
+  },
+  {
+    "title": "¿Permitir que la app %@ también use tu ubicación aunque no la estés utilizando?",
+    "buttons": [
+      "Permitir"
+    ],
+    "shouldAccept": true
+  },
+  {
+    "title": "¿Activar dictado?",
+    "buttons": [
+      "Ahora no"
+    ],
+    "shouldAccept": true
```

Comment:
We should accept this alert, but "Ahora no" translates as "Not now"

Reply:
We should fix this button at least.
For other comments we need just a small confirmation or explanation.